conf/services/capital-gains-tax-calculator-resident-shares.yml

```diff
@@ -7,33 +7,13 @@ serviceDescription: |
 serviceDomain: www.tax.service.gov.uk
 serviceUrl: /calculate-your-capital-gains/resident/shares/disposal-date
 contactFrontendServiceId: CGT
-complianceStatus: partial
-accessibilityProblems:
-  - users were timed out of the service without warning or the option to continue.
-  - legend did not describe the purpose of the grouped elements.
-  - relationships between content on the check your answers page was not clear.
-  - the error skip link did not function as expected.
-  - some elements present that had a colour used for styling focus indication that did not pass the expected ratio.
-  - heading was not used to introduce content.
-  - text used for the ‘details’ component was not descriptive to screen reader users.
-milestones:
-  - description: "Users were timed out of the service without warning or given the opportunity to extend their time in the service. This doesn’t meet WCAG 2.1 success criteria 2.2.1 (timing Adjustable) and 2.2.6 (timeouts)."
-    date: 2023-12-31
-  - description: "The <legend> did not contain the information related to the group of input fields.The <h1> was presented prior to the <legend> leaving the <legend> containing only the hint text. This doesn’t meet WCAG 2.1 success criteria 1.3.1 (info and relationships)."
-    date: 2023-12-31
-  - description: "The relationship between the content on the ‘Check your answers’ page was not clear, <divs> had been used to display content. This doesn’t meet WCAG 2.1 success criteria 1.3.1 (info and relationships)."
-    date: 2023-12-31
-  - description: "The error skip link did not function as expected, as the ID that the ‘href’ referred to did not exist. This doesn’t meet WCAG 2.1 success criteria 2.4.1 (bypass blocks)."
-    date: 2023-12-31
-  - description: "The colour of the focus indicator used throughout the service failed to meet the expected ratio of 3:1 to pass WCAG 2.1 success criteria 1.4.11 Non-text Contrast AA and may prove difficult for some users to perceive. This doesn’t meet WCAG 2.1 success criteria 1.4.11 (non-text contrast)."
-    date: 2023-12-31
-  - description: "The heading had not been used to introduce the content, the <h1> stated an amount. This doesn’t meet WCAG 2.1 success criteria 2.4.6 (headings and labels)."
-    date: 2023-12-31
-serviceLastTestedDate: 2020-10-22
+complianceStatus: full
 statementVisibility: public
 statementCreatedDate: 2020-09-22
 statementLastUpdatedDate: 2023-01-06
```

Comment:
shouldn't this be update to today also?

Reply:
The date is updated as per the ticket 
<img width="856" alt="Screenshot 2024-03-22 at 13 14 57" src="https://github.com/hmrc/accessibility-statement-frontend/assets/86231805/1f4f88cd-bd09-4169-bac6-68760b1adf41">